src/lib/PostPage.svelte

```diff
@@ -224,28 +233,44 @@
     padding-bottom: 1rem;
     border-bottom: solid 3px $color-background-tertiary;
     width: 100%;
+    display: flex;
+    justify-content: space-between;
+  }
+  @media (max-width: 552px) {
+    .meta-data-line {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: center;
+      gap: 0.5rem;
+    }
+    .author {
+      margin-bottom: 0.3rem;
+    }
+  }
+  @media (min-width: 553px) {
+    .spacer {
+      display: none;
+    }
+  }
+  .spacer {
+    padding-left: 1.5rem;
```

Comment:
This looks to me like you wanted it to be more centered regardless of the length of the author name or updated at?

This is a bit tricky, but I'd recommend a different approach. Here is one example how you could do it:

1. Instead of `display: flex` use `display: grid` and make 3 columns: 1fr, 1rem 1fr in width.
2. Put the spacer in the 1rem area column - author and updated info in the other
3. text center your spacer or put it in a `display: flex;`ing div and center it that way.

This should yield a more consistent result.

That could be hard to implement from the description - if you'd like some code sample let me know.

Reply:
I made an attempt to use this, however, I found the consistency to be the best with the latest commit. I tested with various articles and it seemed to have the desired results I was expecting. When I used the grid for smaller screen sizes I would get wrapping issues. But if you have code samples I'd be happy to look into it more.